test/src/test/java/hudson/model/DescriptorTest.java

```diff
@@ -68,14 +68,14 @@
     @Test public void overriddenId() throws Exception {
         FreeStyleProject p = rule.createFreeStyleProject();
         p.getBuildersList().add(new BuilderImpl("builder-a"));
-        rule.configRoundtrip(p);
+        rule.configRoundtrip((Item)p);
```

Comment:
Why not if this can help, but IIRC (though unsure we should riddle the code with this for a specific IDE thinking about it), this is actually an Eclipse IDE specific issue. 
You're using Eclipse isn't it?